passenger/attributes/daemon.rb

```diff
@@ -21,7 +21,8 @@
 # This is a little broad, but because we expect security group rules to prevent
 # anyone except our trusted ALB from connecting anyway, we don't really care
 # who is able to set X-Forwarded-For headers.
-default[:passenger][:production][:realip_from_cidr] = node.fetch(:cloud).fetch('local_ipv4').split('.')[0..1].join('.') + '.0.0/16'
+#default[:passenger][:production][:realip_from_cidr] = node.fetch(:cloud).fetch('local_ipv4').split('.')[0..1].join('.') + '.0.0/16'
+default[:passenger][:production][:realip_from_cidr] = '172.16.0.0/16'
```

Comment:
I assume `node.fetch(:cloud).fetch('local_ipv4').split('.')[0..1].join('.')` is always 172.16?

Reply:
It always is in our environment.  It could change, didn't find a drop in replacement.  Fetching from :cloud just doesn't work.  Certainly can spend more time trying to replicate the functionality.